testing/adios2/transform/TestBZip2Wrapper.cpp

```diff
@@ -0,0 +1,125 @@
+#include <cstdint>
+#include <iostream>
+#include <numeric> //std::iota
+#include <stdexcept>
+
+#include <adios2.h>
+
+#include <gtest/gtest.h>
+
```

Comment:
@chuckatkins Agree with the test...in that case, bzip2 becomes a public dependency for TestNativeBZip2.cpp?